### ISSUE TITLE
Add expiration date field for certificate

### ISSUE THREAD

**Issue by annezazuu** [woocommerce/sensei-certificates#116]
Requested in 511164-zd-woothemes. 

> Expiration Date: For our client's line of work, they have to renew their certificate either every year, 2 years or 3 years (depending on the course they take). We would make a custom template for different courses and expiration dates but is there a way to set a custom date (one where they did NOT complete the course, but say a year / 2 years / 3 years from that course completion date?) 

Adding in an expiration date that then removes the certificate after a certain period of time would be a neat enhancement particularly for certificates that need to be renewed.

**Comment by PDowney** [comment]
This plugin needs a lot of work and is currently being ignored by the dev team. I'd assume it's dead tbh. I believe they're slowly phasing out Sensei as a whole actually.

I wouldn't hold my breath for enhancements.

**Comment by danjjohnson** [comment]
@PDowney You're right that it needs work, but it's definitely not dead, and we will be releasing an update before too long. Sensei itself is also being actively developed with regular releases as you can see here: https://github.com/Automattic/sensei/releases

**Comment by annezazu** [comment]
Requested 862086-zen

**Comment by aicee** [comment]
5252825-zen:

> There is the code {{completion_date}} in the default fields of the certificate example template.
Is the a code like that for an expiration date?
I need certificates to state something like “Valid until {{two_years_from_completion_date}}”
Where {{two_years_from_completion_date}} is nothing more than the completion date + 2 years.

**Comment by cskindley** [comment]
> @PDowney You're right that it needs work, but it's definitely not dead, and we will be releasing an update before too long. Sensei itself is also being actively developed with regular releases as you can see here: https://github.com/Automattic/sensei/releases

That statement would actually hold some weight if the current core functions in plugin worked as intended for some something other than a marketing tool. If they state it does “x”, “x” should actually work as described in the documentation.
